### PR TITLE
Example/rust wasm

### PR DESCRIPTION
**Pull request by alexUXUI**
TL;DR
Adds a Rust-flavored Wasm example. Run `yarn && yarn start` in the root of `module-federation-examples/rust-wasm` to start.

About:
This PR adds an example that uses Module Federation to share a Rust-flavored Wasm module across webpack builds. It respects the convention of this monorepo where each example can start-up by running `yarn && yarn dev` in the example root directory. 

Notes:
I included a built version of the Wasm module so that users of this example do not need to have the Rust->Wasm toolchain installed in order to run the example. There are instructions for local Rust->Wasm development in the `Local Development` section of the README for users who want to tinker with the example a bit.

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
Yaaaaass
